src/app/pages/FeatureIdxPage/index.jsx

```diff
@@ -0,0 +1,6 @@
+import FeatureIdxPage from './FeatureIdxPage';
+import applyBasicPageHandlers from '../utils/applyBasicPageHandlers';
+
+export default applyBasicPageHandlers({
+  addVariantHandling: false,
```

Comment:
Need to check this - I think we need to remove this (because we copied from IdxPage which did not have variants), but our FIX Pages can have variants

Reply:
As I suspected, we need to set this to true (as per FrontPage) https://github.com/bbc/simorgh/blob/latest/src/app/pages/FrontPage/index.jsx#L5